string filepath
---
flexible_assembly_msgs/Config config